staging/src/k8s.io/client-go/discovery/helper.go

```diff
@@ -114,7 +114,7 @@ func (fn ResourcePredicateFunc) Match(groupVersion string, r *metav1.APIResource
 	return fn(groupVersion, r)
 }
 
-// SupportsAllVerbs is a predicate matching a resource iff all given verbs are supported.
+// SupportsAllVerbs is a predicate matching a resource if all given verbs are supported.
```

Comment:
The meaning of `iff` here is `if and only if`. It seems that`iff` is not a well-known acronym, let's expand it to the full phrase here.